src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -450,28 +450,6 @@ public static EnabledValue ParseEnableValue(string value)
             return EnabledValue.Undefined;
         }
 
```

Comment:
In light of the learnings shared by Andy, I think it would be appropriate to add a comment on why these methods are important from a performance perspective to avoid someone else trying to make the same change in the future.